packages/combo-box/test/scrolling.test.js

```diff
@@ -1,30 +1,19 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync, focusout, isIOS } from '@vaadin/testing-helpers';
+import { fixtureSync, focusout, nextRender } from '@vaadin/testing-helpers';
 import './not-animated-styles.js';
 import '../vaadin-combo-box.js';
 
 describe('scrolling', () => {
   let comboBox, overlay, scroller, input;
 
-  beforeEach(() => {
+  beforeEach(async () => {
     comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
+    await nextRender();
     overlay = comboBox.$.overlay;
     input = comboBox.inputElement;
     scroller = comboBox._scroller;
   });
 
-  afterEach(() => {
-    comboBox.close();
-  });
-
-  (isIOS ? describe : describe.skip)('iOS', () => {
-    it('should have momentum scrolling enabled', () => {
```

Comment:
This test is basically checking a single line of CSS that has a corresponding comment, IMO it's enough:

https://github.com/vaadin/web-components/blob/b4fa5d42bf3b35c36c4c075e79d2a3b2dde4d7b1/packages/combo-box/src/vaadin-combo-box-scroller.js#L34-L35